packages/editor-ui/src/plugins/codemirror/completions/utils.ts

```diff
@@ -212,6 +212,31 @@ export const applyCompletion =
 		view.dispatch(tx);
 	};
 
+export const applyBracketAccess = (key: string): string => {
+	return `['${escapeMappingString(key)}']`;
+};
+
+/**
+ * Apply a bracket-access completion
+ *
+ *  @example `$json.` -> `$json['key with spaces']`
+ *  @example `$json` -> `$json['key with spaces']`
+ */
+export const applyBracketAccessCompletion = (
```

Comment:
Found [this failing case](https://share.cleanshot.com/TYfy988N) where escaping is missing.